spec/requests/banners_spec.rb

```diff
@@ -47,4 +47,28 @@
       end
     end
   end
+
+  context "when a banner has expires_at" do
+    let!(:active_banner) { create(:banner, casa_org: casa_org, expires_at: expires_at) }
+
+    context "when expires_at is after today" do
+      let(:expires_at) { 7.days.from_now }
+
+      it "displays the banner" do
+        sign_in volunteer
+        get casa_cases_path
+        expect(response.body).to include "Please fill out this survey"
+      end
+    end
+
+    context "when expires_at is after today" do
```

Comment:
Minor: I think you mean **before** here.

Reply:
fixed